apps/volontulo/views/organizations.py

```diff
@@ -166,22 +166,17 @@ def organization_view(request, slug, id_):  # pylint: disable=unused-argument
     if request.method == 'POST':
         form = VolounteerToOrganizationContactForm(request.POST)
         if form.is_valid():
-            # send email to first organization user (I assume it's main user)
-            profile = Organization.objects.get(id=id_).userprofiles.all()[0]
             send_mail(
                 request,
                 'volunteer_to_organisation',
-                [
-                    profile.user.email,
-                    request.POST.get('email'),
-                ],
+                [up.user.email for up in org.userprofiles.all()],
                 {k: v for k, v in request.POST.items()},
             )
-            messages.success(request, u'Email został wysłany.')
+            messages.success(request, 'Email został wysłany.')
```

Comment:
I don't undestand the question here, but we support only py3k so there's no need for `u` prefix.